content/docs/practices/rules.md

```diff
@@ -49,70 +49,82 @@ conflicts and give you more useful metrics and alerts.
 
 Aggregating up requests per second that has a `path` label:
 
-```
-instance_path:requests:rate5m =
-  rate(requests_total{job="myjob"}[5m])
-
-path:requests:rate5m =
-  sum without (instance)(instance_path:requests:rate5m{job="myjob"})
+```yaml
+groups:
+- name: path-aggregations
+  rules:
+  - record: instance_path:requests:rate5m
+    expr: rate(requests_total{job="myjob"}[5m])
+  
+  - record: path:requests:rate5m
+    expr: sum without (instance)(instance_path:requests:rate5m{job="myjob"}) 
 ```
 
 Calculating a request failure ratio and aggregating up to the job-level failure ratio:
 
-```
-instance_path:request_failures:rate5m =
-  rate(request_failures_total{job="myjob"}[5m])
-
-instance_path:request_failures_per_requests:ratio_rate5m =
+```yaml
+groups:
+- name: req-failures
+  rules:
+  - record: instance_path:request_failures:rate5m
+    expr: rate(request_failures_total{job="myjob"}[5m])
+    
+  - record: instance_path:request_failures_per_requests:ratio_rate5m
+    expr: |
     instance_path:request_failures:rate5m{job="myjob"}
-  /
+    /
```

Comment:
This is inconsistent with the indentation below, we should keep it all consistent as this file is the primary source for how to format rules.